### ISSUE TITLE
Resolve issues updating to Pact 4.4.x

### ISSUE THREAD

**Issue by holly-cummins** [quarkiverse/quarkus-pact#47]
Pact 4.4.x introduces a few problems: 

- @knutwannheden observes that 4.4.0 introduces Groovy 4, and this may be problematic when used together with other Groovy-based frameworks like Rest-Assured for example.
- Although 4.4.1 and 4.4.2 built cleanly, 4.4.3 introduces a test failure (see #45):

```
2023-01-05 18:24:15,719 ERROR [io.qua.test] (Test runner thread) >>>>>>>>>>>>>>>>>>>> Summary: <<<<<<<<<<<<<<<<<<<<
FarmContractTest#pactVerificationTestTemplate(PactVerificationContext) java.lang.NoClassDefFoundError: org/apache/commons/text/translate/AggregateTranslator
2023-01-05 18:24:15,721 ERROR [io.qua.test] (Test runner thread) >>>>>>>>>>>>>>>>>>>> 0 TESTS FAILED <<<<<<<<<<<<<<<<<<<<
0 tests failed (1 passing, 0 skipped), 1 test was run in 4544ms. Tests completed at 18:24:15.
```
This may be caused by a new dependency which needs to be declared parent first.

**Comment by gastaldi** [comment]
@all-contributors add @holly-cummins for code and maintenance

**Comment by holly-cummins** [comment]
The good news here is that Quarkus used to enforce Groovy 3 versions and is now enforcing the latest Groovy 4 version. This has caused a failure when we build this extension against the `HEAD` Quarkus stream (see, for example, https://github.com/quarkiverse/quarkus-pact/actions/runs/4239609215/jobs/7367826379) and https://github.com/quarkiverse/quarkiverse/issues/94#issuecomment-1437918108. However, we can fix the ecosystem CI by accepting #79 (along with a bit of extra parent first configuration). I'll prepare a PR which combines the pact version change and the quarkus version change.
